### PR TITLE
Chore add onboarding role

### PR DESCRIPTION
**Pull request by cameronperera**
### Screenshots (if appropriate):
Screen Shot 2022-06-16 at 4 39 56 PM

### CODE REVIEW

File: addons/core/translations/en-us.yaml [hunk 1]
```diff
@@ -77,10 +77,12 @@ onboarding:
       desktop: Using Boundary Desktop
       developers: Developers
       controller: Controller URL
+      role: We have created your scopes and target
```
Comment (randallmorey) [inline addons/core/translations/en-us.yaml:80]: This key/value mapping doesn't make sense to me.  Why is the key "role"?

File: ui/admin/app/templates/onboarding/quick-setup/create-resources/success.hbs [hunk 2]
```diff
@@ -5,53 +5,60 @@
       <h2>{{t 'onboarding.success.titles.connect'}}</h2>
     </:header>
     <:body>
-      <div class='onboarding-quick-setup-success-boundary'>
-        <div class='onboarding-quick-setup-success-sub-title'>
-          <Rose::Icon @name='logo-app-background' @size='extra-large' />
+      <div class='onboarding-quick-setup-success-role'>
+        <span>
```
Comment (randallmorey) [inline ui/admin/app/templates/onboarding/quick-setup/create-resources/success.hbs:9]: What is the purpose of this `span`?

**Review by randallmorey — APPROVED** [review]

File: ui/admin/app/templates/onboarding/quick-setup/create-resources/success.hbs (hunk 2 above)
Comment (cameronperera) [reply ui/admin/app/templates/onboarding/quick-setup/create-resources/success.hbs:9]: To make the Icon and h3 tag inline with each other. That is what line 300 in `app.scss` is for.

File: addons/core/translations/en-us.yaml (hunk 1 above)
Comment (cameronperera) [reply addons/core/translations/en-us.yaml]: Updated to `creation-success`. Even that seems not exactly right but more accurate than "role".

**Review by DhariniJeeva — APPROVED** [review]
ty! LGTM!

**Pull request merged**